tutorials/CommandsArguments.md

```diff
@@ -0,0 +1,64 @@
+
+Unlike normal arguments, a repeating one **may not** return a single value but multiple, all of the same type (or of the first one that resolves if using multiple possibles), that being said, it's advisable to use the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator) for arrays if you have multiple arguments.
+
+> **Note**: A common mistake is to use `[...choices]` in the `Command#run`, as you may know, `[...choices]` is identical to `choices` by the simple fact you're creating an array with all the elements of choices, or said in another way: an identical array. You can check it in the MDN link for spread syntax.
+
+## Possible Structure
+
+As mentioned before, a `Tag` is a **parsed argument** that contains an array of **multiple Possible**s. But what is their format? Generally, they're written as `name:type`, where `name` is the name of the argument, and `type` is the type. However, depending on the type you can use the options or not.
+
+- **name** is the name of the argument, they're used to identify an argument by giving it a name, a good name should be self explanatory and easy to read for the end user.
+- **type** is the name of the type, they're not arbitrary but depends on the methods {@link ArgResolver} has. By default, there are many of them, such as `string`, `integer`, `msg`, `user`, `member`. You can also extend ArgResolver to implement extra methods, check this tutorial: {@tutorial CreatingCustomArguments}, however, if no type is given, the parser will assume them as **literal**.
+- **options** always have the format `{min,max}`, taking two integers, both optional (you can specify `{,max}` for an argument with no minimum, or `{min,}` for an argument with a minimum but not a maximum), and they're used mostly in the `string`, `number` (float) and `integer`, being the string length and the number the variables to handle, respectively.
+
+## Examples
+
+- We want a required argument that takes a user, and an optional argument for a role: `<targetUser:user> [targetRole:role]`.
+- We want to accept `set`, `add`, `remove` or `reset` as required argument: `<set|add|remove|reset>`. (Remember that if no type is given, Usage will imply they are literals).
+- We want to get a GuildMember instance, or its user ID if the user is not in the guild: `<targetMember:member|string{17,18}>`. (Either a GuildMember resolvable or a string with a length of 17-18, which matches with a Discord snowflake's length).
```

Comment:
the second possible is incomplete, it doesn't have it's name property. This will error in the console.